js/output/shared/loop-protect.js

```diff
@@ -71,7 +72,7 @@ window.LoopProtector.prototype = {
         } else if (now - this.branchStartTime > this.timeout) {
             if (this.visible) {
                 if (!this.reportLocation) {
-                    let error = new Error("KA_INFINITE_LOOP");
+                    error = new Error("KA_INFINITE_LOOP");
```

Comment:
I can also change both to const instead of introducing a let above, either way works.

Reply:
Let's change both to `const`.